src/onelogin/saml2/idp_metadata_parser.py

```diff
@@ -148,7 +148,7 @@ def parse(
 
                 idp_entity_id = entity_descriptor_node.get('entityID', None)
 
-                want_authn_requests_signed = entity_descriptor_node.get('WantAuthnRequestsSigned', None)
+                want_authn_requests_signed = idp_descriptor_node.get('WantAuthnRequestsSigned', None)
```

Comment:
Note that this causes a regression, because `idp_descriptor_node.get('WantAuthnRequestsSigned', None)` returns the string `'false'` for my config (and probably others as well) which gets later evaluated to `True` in https://github.com/SAML-Toolkits/python3-saml/blob/538622df70b45520f60e7fb6b569aa00be54f504/src/onelogin/saml2/settings.py#L480